src/main/java/com/example/enterprise/service/IExpenseService.java

```diff
@@ -19,7 +19,7 @@
      * Return all saved expenses.
      * @return a collection of all expense entries.
      */
-    List<Expense> showAll();
+    List<Expense> showAllExpenses();
```

Comment:
Changed the method name from showAll() to showAllExpenses() because showAll doesn't specify what to show. An outsider would not understand what it is trying to show. showAllExpenses() name is much more meaningful and provides more clarity to the code logic.